commit 4add6942e51ec39a3f9f183700b6bcc3669629d1
Author: Heiko Carstens <heiko.carstens@de.ibm.com>
Date:   Mon Jul 2 10:54:02 2018 +0200

    s390/sysinfo: add missing #ifdef CONFIG_PROC_FS
    
    [ Upstream commit 9f35b818a2f90fb6cb291aa0c9f835d4f0974a9a ]
    
    Get rid of this compile warning for !PROC_FS:
    
      CC      arch/s390/kernel/sysinfo.o
    arch/s390/kernel/sysinfo.c:275:12: warning: 'sysinfo_show' defined but not used [-Wunused-function]
     static int sysinfo_show(struct seq_file *m, void *v)
    
    Signed-off-by: Heiko Carstens <heiko.carstens@de.ibm.com>
    Signed-off-by: Martin Schwidefsky <schwidefsky@de.ibm.com>
    Signed-off-by: Sasha Levin <alexander.levin@microsoft.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/arch/s390/kernel/sysinfo.c b/arch/s390/kernel/sysinfo.c
index 54f5496913fa..12f80d1f0415 100644
--- a/arch/s390/kernel/sysinfo.c
+++ b/arch/s390/kernel/sysinfo.c
@@ -59,6 +59,8 @@ int stsi(void *sysinfo, int fc, int sel1, int sel2)
 }
 EXPORT_SYMBOL(stsi);
 
+#ifdef CONFIG_PROC_FS
+
 static bool convert_ext_name(unsigned char encoding, char *name, size_t len)
 {
 	switch (encoding) {
@@ -301,6 +303,8 @@ static int __init sysinfo_create_proc(void)
 }
 device_initcall(sysinfo_create_proc);
 
+#endif /* CONFIG_PROC_FS */
+
 /*
  * Service levels interface.
  */
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1144_4add6942e51ec39a3f9f183700b6bcc3669629d1.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1144_4add6942e51ec39a3f9f183700b6bcc3669629d1.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
